crates/sui-graphql-rpc/src/types/object.rs

```diff
@@ -987,6 +988,19 @@ impl Object {
             }),
         }
     }
+
```

Comment:
nit: Can we move this constructor next to the other constructor `from_native`, at the top of this `impl` block?

Reply:
(although now, looking at the overall order of these functions, it is somewhat a mess so this may be a bit of a band-aid).